copter/source/docs/frame-type-configuration.rst

```diff
@@ -21,6 +24,22 @@ Next select the frame "Type" for your vehicle. The default type is **X**.
 For Tricopters, Y6, Traditional Helicopters, Bicopters, SingleCopters and CoaxCopters the frame type is ignored.
 
 Motor Order Diagrams
-====================
+--------------------
 
 See the :ref:`Connect ESCs and Motors <connect-escs-and-motors>` page for the list of supported multicopter frames
+
+Traditional Helicopters
+=======================
+
+Traditional Helicopters run a different version of firmware. See :ref:`traditional-helicopters` for setup and tuning information.
+
+Other Configurations
+====================
+
+.. toctree::
+   :maxdepth: 1
+
+   Tricopter <tricopter>
+   SingleCopter and CoaxCopter <singlecopter-and-coaxcopter>
+   HeliQuads <heliquads>
```

Comment:
HeliQuads could also be linked from the Traditional Helicopter section under "Dual Helicopters".  It's basically a hybrid frame so it could be linked from both areas.. although it's not a particularly popular frame so maybe it doesn't matter.

Reply:
will add link in Helicopter

Reply:
since it uses  trad heli it should be under it...